src/lib/Scenes/Partner/Components/PartnerShows.tsx

```diff
@@ -12,34 +13,42 @@ import { PartnerShowsRailContainer as PartnerShowsRail } from "./PartnerShowsRai
 
 const PAGE_SIZE = 6
 
-const ShowGridItem = ({
-  show,
-  itemIndex,
-}: {
+interface ShowGridItemProps {
   show: PartnerShows_partner["pastShows"]["edges"][0]["node"]
   itemIndex: number
-}) => {
-  const showImageURL = show.coverImage && show.coverImage.url
-  const styles = itemIndex % 2 === 0 ? { paddingRight: space(1) } : { paddingLeft: space(1) }
-  return (
-    <GridItem key={show.id}>
-      <TouchableWithoutFeedback onPress={null}>
-        <Box style={styles}>
-          {showImageURL ? (
-            <BackgroundImage key={show.id} style={{ resizeMode: "cover" }} source={{ uri: showImageURL }} />
-          ) : (
-            <EmptyImage />
-          )}
-          <Spacer mb={0.5} />
-          <Sans size="2">{show.name}</Sans>
-          <Serif size="2" color="black60">
-            {show.exhibitionPeriod}
-          </Serif>
-        </Box>
-      </TouchableWithoutFeedback>
-      <Spacer mb={2} />
-    </GridItem>
-  )
+}
+
+class ShowGridItem extends React.Component<ShowGridItemProps> {
+  onPress = () => {
+    const { show } = this.props
+    SwitchBoard.presentNavigationViewController(this, `/show/${show.slug}`)
+  }
+
+  render() {
+    const { show, itemIndex } = this.props
+    const showImageURL = show.coverImage && show.coverImage.url
+    const styles = itemIndex % 2 === 0 ? { paddingRight: space(1) } : { paddingLeft: space(1) }
```

Comment:
I know this is unrelated to this change, but I recall discussing (with @alloy?) that we prefer to stick with margins, rather than padding. Or I might be mis-remembering?

Reply:
In some cases they can be used interchangeably as long as the padding exists on the wrapper, which is the case here. Having a blanket rule about using padding instead of margin or vice versa seems problematic, they both have their uses and I think it can just come down to personal preference.

Reply:
Cool, thanks for the explainer :bow:

Reply:
What we discussed is that components should not define whitespace around themselves, but leave that up to the parent composing it. Otherwise you inevitably end-up battling styles.